### ISSUE TITLE
How to use tensorboard to show training and validations line ?(how to know how many training steps I must to set?)

### ISSUE THREAD

**Issue by sed0724963** [tensorflow/models#6190]
I use the" tensorflow object detection api "to training a detection model.
and I use the VOC format data,model is faster rcnn resnet 101 coco.

the training config and train.py is from the samples.
URL:https://github.com/tensorflow/models/tree/master/research/object_detection

SO,I use the recommended directory structure to run tensorboard,command line is tensorboard --logdir=${MODEL_DIR}.

it's output is this image:it have many loss function.:have total loss,clone loss...about six,but it's only have one lines on every image.
<img width="1025" alt="2019-02-12 6 44 02" src="https://user-images.githubusercontent.com/38509051/52630230-ab2a0100-2ef6-11e9-9c74-912ccc9f60e5.png">

 I want to show train and validation line on same image,like this:it's have two lines(train and validation) on same image.
<img width="777" alt="2019-02-12 6 47 15" src="https://user-images.githubusercontent.com/38509051/52630321-e6c4cb00-2ef6-11e9-9d00-dde29b79f23e.png">

So, how can I sucessful to do this??
the most teaching from Internet are all use mnist,and I don't understand mnist.
the tensorboard code I assumed it's on trainer.py,but I don't how to modify it to become my want.

------------------------

### System information
- **What is the top-level directory of the model you are using**: 
../models/research
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
NO
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
windows 10
- **TensorFlow installed from (source or binary)**:
pip
- **TensorFlow version (use command below)**:
tensorflow 1.8-gpu
- **Bazel version (if compiling from source)**:
I don't know
- **CUDA/cuDNN version**:
cuda 9.0, cnDNN7.1.4
- **GPU model and memory**:
ASUS GeForce GTX1080 8G
- **Exact command to reproduce**:
I don't know

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
